src/styles/mixins/_tables.scss

```diff
@@ -236,20 +238,135 @@
             &.row-link {
               padding: var(--kui-space-0, $kui-space-0);
 
+              .cell-wrapper {
+                padding: var(--kui-space-50, $kui-space-50) var(--kui-space-60, $kui-space-60);
+              }
+
               a.cell-wrapper {
                 color: var(--kui-color-text, $kui-color-text);
                 display: block;
-                padding: var(--kui-space-50, $kui-space-50) var(--kui-space-60, $kui-space-60);
                 text-decoration: none;
               }
+
+              .expandable-row-control-container {
+                padding: var(--kui-space-50, $kui-space-50) var(--kui-space-60, $kui-space-60);
+              }
+            }
+
+            .expandable-row-control-container {
+              display: flex;
+
+              .expandable-row-control {
+                @include defaultButtonReset;
+
+                color: var(--kui-color-text-neutral, $kui-color-text-neutral);
+
+                &:hover,
+                &:focus,
+                &:focus-visible {
+                  color: var(--kui-color-text-neutral-strong, $kui-color-text-neutral-strong);
+                }
+
+                &:focus-visible {
+                  box-shadow: var(--kui-shadow-focus, $kui-shadow-focus);
+                }
+
+                .expandable-row-control-icon {
+                  height: $tableCaretIconSize !important;
+                  transition: transform $kongponentsTransitionDurTimingFunc;
+                  width: $tableCaretIconSize !important;
+                }
+
+                &.expanded {
+                  .expandable-row-control-icon {
+                    transform: rotate(90deg);
+                  }
+                }
+              }
+            }
+          }
+
+          &.last-row {
+            border-bottom: none;
+          }
+
+          &.expandable-content-row {
+            background-color: var(--kui-color-background-neutral-weakest, $kui-color-background-neutral-weakest);
+            height: auto;
+
+            > td {
+              padding: var(--kui-space-50, $kui-space-50) var(--kui-space-0, $kui-space-0)
+                var(--kui-space-50, $kui-space-50) var(--kui-space-60, $kui-space-60);
```

Comment:
Should this apply to `.expandable-content-wrapper` so that it won't produce a gap above the scrollbar?

<img width="306" alt="image" src="https://github.com/user-attachments/assets/cf32c3e5-7ccc-4fcd-a327-98fe40e106e2">

Reply:
It seems to be inconsistent with how we are dealing with scrollable containers with `maxHeight` in other components like `<CodeBlock>` or `<Modal>`?

Reply:
Asked the team in the thread whether we have a preference.

Reply:
We decided to leave this as is for now